src/cljs/main/broadfcui/components/buttons.cljs

```diff
@@ -19,7 +19,9 @@
         (map? disabled?)
         (@modal-constructor
          (if (= (:type disabled?) :error) :error :message)
-         (assoc disabled? :on-dismiss on-dismiss))))
+         (assoc disabled? :on-dismiss on-dismiss))
+        :else
```

Comment:
this else catches divs (and other things...)...

is there a way to test for a div specifically?

Reply:
It also catches `false`/`nil`. This function won't get called when `disabled?` is falsy, but it still feels wrong.

There's a `renderable?` function in components.cljs, which you could put in place of the `string?` check above.